### PR TITLE
feat: Add first-steps content and style

### PR DESCRIPTION
**Pull request by Carlos-Muniz**
A module with the first steps has been added via first-steps.rst. This
is references in index.rst. To add better style, the edx-theme has been
added in conf.py.

### CODE REVIEW

File: docs/source/first-steps.rst [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+Work email
+----------
+Look for an email asking you to change your password that mentions “the Center
+for Reimagining Learning”. Submit a new password, and sign into your now
+accessible account.
+
+Jira/Confluence
+---------------
+Set up an account on Jira/Confluence via Google using your work email address.
```
Comment (feanil) [inline docs/source/first-steps.rst]: Make `Jira/Confluence` a link to the page where you should go to register a new account.

File: docs/source/first-steps.rst [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+-------------------
+Set up an account on discuss.openedx.org by selecting the Google Account option
+using the Sign Up button in the upper right. Use a tcril.org email address or
+SSO.
+
+GitHub
+------
+Using your personal account for work now? Better change the email too. You will
+be getting many notifications a day and you’ll want them all in one place. Use
+the settings under email and notifications in Github to add your email address
```
Comment (feanil) [inline docs/source/first-steps.rst]: Link to this settings page.

**Review by feanil — APPROVED** [review]
Some small suggestions.

Meta Review: Where possible, provide links in the doc so people don't have to go hunting for them.

Once links are added, this looks good to me, feel free to merge if the docs are building at that point.

File: docs/source/conf.py [hunk 3]
```diff
@@ -32,6 +33,7 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
+    'edx_theme'
```
Comment (feanil) [inline docs/source/conf.py]: Given that the edx theme is not actually the right theme for us, let's just not use it from the start.  If we don't like the default, let's figure out what we need from a theme and try to use one off the shelf if we can.

See https://openedx.slack.com/archives/C02M89BSV9A/p1639155646062400 for more context.

**Review by feanil — CHANGES_REQUESTED** [review]
Changed my mind, let's drop the edx_theme for now.

File: docs/source/first-steps.rst (hunk 1 above)
Comment (sarina) [reply docs/source/first-steps.rst]: Also should you log in with SSO or with a uname/pwd? (pretty sure it's SSO)

File: docs/source/first-steps.rst [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+and change where the notifications go. Be sure to enable 2 Factor 
+Authorization.
```
Comment (sarina) [inline docs/source/first-steps.rst]: s/Authorization/Authentication/

**Review by sarina — CHANGES_REQUESTED** [review]

File: docs/source/first-steps.rst (hunk 1 above)
Comment (Carlos-Muniz) [reply docs/source/first-steps.rst]: The [link to sign up](https://id.atlassian.com/signup?continue=https%3A%2F%2Fopenedx.atlassian.net%2Flogin%3FredirectCount%3D1%26dest-url%3Dhttps%253A%252F%252Fopenedx.atlassian.net%252Fjira%252Fprojects%26application%3Djira&application=jira) is very long compared to the [link to the main page](https://openedx.atlassian.net/jira/projects), so I was thinking of using the second link and adding instructions on where to sign in.

Comment (Carlos-Muniz) [reply docs/source/first-steps.rst]: I've added the links

File: docs/source/first-steps.rst (hunk 2 above)
Comment (Carlos-Muniz) [reply docs/source/first-steps.rst]: Settings have been linked

File: docs/source/conf.py (hunk 3 above)
Comment (Carlos-Muniz) [reply docs/source/conf.py]: I've removed the edx-theme and replaced it with the rtd-theme which supports custom colors and custom logo.

File: docs/source/first-steps.rst (hunk 4 above)
Comment (Carlos-Muniz) [reply docs/source/first-steps.rst]: I've added SSO to the JIRA instructions and replace Authorization with Authentication

**Review by feanil — APPROVED** [review]

**Review by sarina — APPROVED** [review]

**Pull request merged**